common/app/views/fragments/containers/facia_cards/standardContainer.scala.html

```diff
@@ -39,7 +39,7 @@
          data-id="@containerDefinition.dataId">
 
         @for(sliceWithCards <- containerLayout.slices) {
-            @slice(sliceWithCards, containerDefinition.index, isElectionContainer = containerDefinition.displayName.contains("general election 2017"), frontProperties = Some(frontProperties))
```

Comment:
I had forgotten to remove a `)` here which broke things. This has now been properly removed.